diktat-gradle-plugin/src/main/kotlin/org/cqfn/diktat/plugin/gradle/Utils.kt

```diff
@@ -2,39 +2,12 @@
  * Utilities for diktat gradle plugin
  */
 
+@file:Suppress("FILE_NAME_MATCH_CLASS")
+
```

Comment:
Do we still need it here?

Reply:
Yes, there is a single class in this file, but I don't want to change file name accordingly.